app/web/js/localeSwitch.js

```diff
@@ -41,7 +56,7 @@ window.addEventListener('load', function () {
                             const optionElement = optionElements[k];
                             if (optionElement.value === choice) {
                                 optionElement.setAttribute('selected', 'selected');
-                                return true;
+                                continue;
```

Comment:
Is this change in behavior intended; Breaking out of the function VS continuing it's routine? 

The way I see it, it now adds the selected class and then removes it right afterwards. But I have not studied this code in super great detail. Just noticing a little smell. Please elaborate.

Reply:
It's intentional.  

What it does is one of two things:
- it sets the selected attribute if the value of the element = the choice of the user.  In that case the continue ensures that only the if statement is executed in that iteration of the for loop (which means the selected attribute isn't removed again just after setting it.
- it removes the selected attribute in case the value does not correspond with the choice of the user.  This is necessary (and is what didn't happen before), because one of the option-elements has the selected class given that it corresponds to the current value, rather than the value the user desires.

Reply:
Allright no objections from my side then! Thanks for making things clear.